lib/fluent/plugin/out_buffered_stdout.rb

```diff
@@ -0,0 +1,38 @@
+#
+# Fluentd
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+require 'fluent/output'
+
+module Fluent
+  class BufferedStdoutOutput < BufferedOutput
```

Comment:
Strictly speaking:
* There's no need to implement `format` method if the plugin is a subclass of `ObjectBufferedOutput`
  * `ObjectBufferedOutput` plugins do NOT call `format` in its `emit`, and write msgpack binaries into buffer directly
  * https://github.com/fluent/fluentd/blob/master/lib/fluent/output.rb#L414
  * its chunks can be iterated using `chunk.msgpack_each` in `write_objects`, just like `out_forward` is doing
  * https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin/out_forward.rb#L197
* It's needed to implement `format` method if a plugin is a subclass of `BufferedOutput`
  * BufferedOutput plugins write return values of `format` into chunks
  * Return values of `format` method MUST be msgpack binaries if you want to do `msgpack_each` for chunks